doc/qbittorrent-nox.1.md

```diff
@@ -30,6 +30,12 @@ the default account user name is "admin" with "adminadmin" as a password.
 
 
 # OPTIONS
+
+TODO: in addition to basically copy pasting the other manpage:
```

Comment:
TODO shouldn't appear in man pages, unless this is only temporary.

Reply:
I know. The point of the TODOs is to remind of what has to be done and for you and others to suggest what to do.